packages/central-server/src/apiV2/surveyResponse.js

```diff
@@ -29,7 +29,6 @@ const createSurveyResponseValidator = models =>
     survey_id: [hasContent, constructRecordExistsWithId(models.survey)],
     start_time: [constructIsEmptyOr(takesDateForm)],
     end_time: [constructIsEmptyOr(takesDateForm)],
-    timestamp: [hasContent],
```

Comment:
constructIsEmptyOr(takesDateForm) ?

Reply:
Good call, will do :+1:

Reply:
Discovered that this value can actually be a iso-string or a number. Did a bit of a spike into working out if we could deprecate it entirely, but I don't think we can. It's used as part of the import logic as well as being called externally by msupply.

I don't think it's possible for our ObjectValidator to do a `isANumber OR takesDateForm` so I might just leave it with no validation

Reply:
All good!